lblod/frontend-form-builder
# SFB-118: translations

## ID
 [SFB-118](https://binnenland.atlassian.net/browse/SFB-118)

 ## Description

In the form builder a mix of dutch and english is used atm. Change everything to the ~english~ Dutch translation. As we are already translating i added a translation package to the project so we can find almost all of the translations in one place.

 ## Type of change


 ## How to test

 1. See if everything is translated to Dutch
 2. Some things could not be translated because of it is set in a component (form overview table)

## Todo

- [ ] translate the ttl values

 ## Links to other PR's

 - /
